src/Pages/Licensing.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import BackBtn from "../components/BackBtn/BackBtn";
+
+
+const Licensing = () => {
+  return (
+
+    <>
+    <div className="py-8 mb-5 w-[50vw] justify-start ml-auto mr-auto mt-10">
+    <div>
+      <BackBtn Page={"Licensing"} />
+      <LicenseWrapper>
+    
+      <section>
+      <p>Last updated: May 17, 2024</p>
+        <p>Welcome to NactoCare. This Licensing Agreement governs your access and use of our software, content, and services. By accessing or using any part of our services, you agree to comply with and be bound by the terms outlined in this agreement.</p>
+      </section>
+      <section>
+        <h2>License Grant</h2>
+        <p>NactoCare hereby grants you a limited, non-exclusive, non-transferable, and revocable license to access and use our services strictly in accordance with the terms of this agreement. This license does not include any rights to:</p>
```

Comment:
We use GNU GPL 3 Public license and thus we don't limit the use or distribution of software or service. I would suggest to change the licensing based on what GNU GPL 3.0 provides.